File: billing_ui_app/src/app/core/models/billing/payment-summary.ts
import {PaymentPostingMethods} from "@core/models/billing/payment-posting-methods";

export interface PaymentSummary extends UpdateManualPaymentSummary{    
    paymentAmount: number;
    postedAmount: number;
    remainingAmount: number;
    funderName: string;
    payee: string;
    paymentMethodId: number;
    isManual: boolean;
    paymentTypeId: number;
} 

export interface UpdateManualPaymentSummary {
    id: number;
    postDate: Date|undefined;
    paymentMethodId: number;
    paymentMethod: string;
    paymentMethodEntity: PaymentPostingMethods;
    depositDate: Date|undefined;
    referenceNumber: string;
    paymentAmount?: number;
}

File: Microservices/BillingService/BillingService.Domain/Models/PaymentPosting/PaymentSummary.cs
using System;

namespace BillingService.Domain.Models.PaymentPosting
{
    public class PaymentSummary
    {
        public int Id { get; set; }
        public decimal PaymentAmount { get; set; }
        public decimal PaymentAmountOrig { get; set; }
        public decimal PostedAmount { get; set; }
        public decimal RemainingAmount { get; set; }
        public DateTime? PostDate { get; set; }
        public DateTime? DepositDate { get; set; }
        public string FunderName { get; set; }
        public string PaymentMethod { get; set; }
        public int PaymentMethodId { get; set; }
        public string Payee { get; set; }
        public string ReferenceNumber { get; set; }
        public bool IsManual { get; set; }

        public int PaymentTypeId { get; set; }

    }
}